features/SideNavigation.feature

```diff
@@ -0,0 +1,165 @@
+@owner:lilla
+@author:lilla
+
```

Comment:
Note that when QA is run you will want to include some of the personalization specs as well since that aspect of the sidebar behaviour is not mentioned here

Reply:
I am planning to include the whole Personalization.feature file